src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -111,10 +129,36 @@ private EppoHttpClient buildHttpClient(
   }
 
   protected void loadConfiguration() {
-    requestor.load();
+    configurationWrangler.load();
   }
 
-  // TODO: async way to refresh for android
+  /**
+   * Loads the configuration asynchronously, off-thread
+   *
+   * @param skipCache This allows the Configuration Wrangler to skip trying to load from the cache.
+   *     This is useful when we want only the freshest configuration.
+   * @param callback Called when load is complete.
+   */
+  protected void loadConfigurationAsync(
+      boolean skipCache, @Nullable InitializationCallback callback) {
+    configurationWrangler.loadAsync(
+        skipCache,
+        new ConfigurationWrangler.LoadCallback() {
```

Comment:
these two callbacks actually have the same shape.
`InitializationCallback`, however, is part of this package's protected API via this method, so it inclined me to separate the two.